src/test/java/org/junitpioneer/testkit/PioneerTestKit.java

```diff
@@ -38,12 +41,21 @@ public static ExecutionResults executeTestMethod(Class<?> testClass, String test
 	 *
 	 * @param testClass Name of the test class
 	 * @param testMethodName Name of the test method (of the given class)
-	 * @param methodParameterTypes Full qualified types names of the parameters (e.g. "java.nio.file.Path")
+	 * @param methodParameterTypes Class type(s) of for the parameters
 	 * @return The execution results
+	 * @throws IllegalArgumentException when methodParameterTypes is null
```

Comment:
But why only for that one parameter?

Reply:
Because the other ones are handled by the Jupiter TestKit and I want to avoid NPE in Pioneers code (accessing the get name in the stream).